### ISSUE TITLE
[1.4.x] Trying to install gives namespace error

### ISSUE THREAD

**Issue by Portugao** [Guite/MostGenerator#669]
```
  in vendor/doctrine/orm/lib/Doctrine/ORM/ORMException.php at line 236  -

         */
        public static function unknownEntityNamespace($entityNamespaceAlias)
        {
            return new self(
                "Unknown Entity namespace alias '$entityNamespaceAlias'."
            );
        }

at ORMException ::unknownEntityNamespace ('MUMUPollsModule')
in vendor/doctrine/orm/lib/Doctrine/ORM/Configuration.php at line 194  +

at Configuration ->getEntityNamespace ('MUMUPollsModule')
in vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php at line 554  +

at ClassMetadataFactory ->getFqcnFromAlias ('MUMUPollsModule', 'PollEntity')
in vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/AbstractClassMetadataFactory.php at line 194  +

at AbstractClassMetadataFactory ->getMetadataFor ('MUMUPollsModule:PollEntity')
in vendor/doctrine/orm/lib/Doctrine/ORM/EntityManager.php at line 265  +

at EntityManager ->getClassMetadata ('MUMUPollsModule:PollEntity')
in vendor/doctrine/orm/lib/Doctrine/ORM/Repository/DefaultRepositoryFactory.php at line 67  +

at DefaultRepositoryFactory ->createRepository (object(EntityManager), 'MUMUPollsModule:PollEntity')
in vendor/doctrine/orm/lib/Doctrine/ORM/Repository/DefaultRepositoryFactory.php at line 50  +

at DefaultRepositoryFactory ->getRepository (object(EntityManager), 'MUMUPollsModule:PollEntity')
in vendor/doctrine/orm/lib/Doctrine/ORM/EntityManager.php at line 665  +

at EntityManager ->getRepository ('MUMUPollsModule:PollEntity')
in modules/MU/mupolls-module/Base/MUPollsModuleInstaller.php at line 373  +

at MUPollsModuleInstaller ->createDefaultData (array())
in modules/MU/mupolls-module/Base/MUPollsModuleInstaller.php at line 61  +

at MUPollsModuleInstaller ->install ()

at call_user_func (array(object(MUPollsModuleInstaller), 'install'))
in system/Zikula/Module/ExtensionsModule/Api/AdminApi.php at line 1036  +

at AdminApi ->initialise (array('id' => '18', 'interactive_init' => false))

at call_user_func (array(object(AdminApi), 'initialise'), array('id' => '18', 'interactive_init' => false))
in lib/util/ModUtil.php at line 1155  +

at ModUtil ::exec ('ZikulaExtensionsModule', 'admin', 'initialise', array('id' => '18', 'interactive_init' => false), true, null)
in lib/util/ModUtil.php at line 1231  +

at ModUtil ::apiFunc ('ZikulaExtensionsModule', 'admin', 'initialise', array('id' => '18', 'interactive_init' => false))
in system/Zikula/Module/ExtensionsModule/Controller/AdminController.php at line 633  +

at AdminController ->initialiseAction (object(Zikula_Request_Http))

at call_user_func_array (array(object(AdminController), 'initialiseAction'), array(object(Zikula_Request_Http)))
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php at line 145  +

at HttpKernel ->handleRaw (object(Zikula_Request_Http), '1')
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php at line 66  +

at HttpKernel ->handle (object(Zikula_Request_Http), '1', true)
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php at line 64  +

at ContainerAwareHttpKernel ->handle (object(Zikula_Request_Http), '1', true)
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php at line 185  +

at Kernel ->handle (object(Zikula_Request_Http))
in index.php at line 24  +

```

**Comment by Portugao** [comment]
The tables get created..state of the module is '1', not installed.

**Comment by Guite** [comment]
@cmfcmf any idea why this does not affect the routes module? did you change something in the generated code with regards to this?

**Comment by cmfcmf** [comment]
> @cmfcmf any idea why this does not affect the routes module? did you change something in the generated code with regards to this?

No, I don't remember changing something.

**Comment by Guite** [comment]
Maybe this is also related to the module location (`modules/MU/mupolls-module/`)

**Comment by Portugao** [comment]
I tried different ways to position the module with no helping effect?!

**Comment by Guite** [comment]
Can you please try `modules/MUMUPollsModule/`?

**Comment by Portugao** [comment]
Same issue!

**Comment by Guite** [comment]
Please try the following:

Open `modules/MU/mupolls-module/Base/MUPollsModuleInstaller.php` and navigate to line 373. There should be something like
```php
 protected function createDefaultData($categoryRegistryIdsPerEntity)
{
    $entityClass = 'MUMUPollModule:PollEntity';
    $this->entityManager->getRepository($entityClass)->truncateTable();
}
```

Please change that to
```php
 protected function createDefaultData($categoryRegistryIdsPerEntity)
{
    $entityClass = 'MU\MUPollModule\Entity\PollEntity';
    $this->entityManager->getRepository($entityClass)->truncateTable();
}
```
(repeat that for all entities you had defined in your model)

Afterwards try installing it again in a fresh core installation.

I assume that the shorthand syntax is not available as long as the module has not been fully installed yet.

**Comment by Portugao** [comment]
Then the module get installed, leading to this url:
```
domain//extensions/modules?startnum=0&state=0&postinstall=[21]#traces-link-1
```
and these errors:
```
 Runtime Notice: Declaration of MU\MUPollsModule\Controller\AjaxController::getItemListFinderAction() should be compatible with MU\MUPollsModule\Controller\Base\AjaxController::getItemListFinderAction(Symfony\Component\HttpFoundation\Request $request) in domain/modules/MU/mupolls-module/Controller (which is being imported from "/domain/modules/MU/mupolls-module/Resources/config/routing.yml").
500 Internal Server Error - FileLoaderLoadException
1 linked Exception:

    ContextErrorException »

```
and further:
```
 FileLoaderLoadException: Runtime Notice: Declaration of MU\MUPollsModule\Controller\AjaxController::getItemListFinderAction() should be compatible with MU\MUPollsModule\Controller\Base\AjaxController::getItemListFinderAction(Symfony\Component\HttpFoundation\Request $request) in domain/modules/MU/mupolls-module/Controller (which is being imported from "domain/modules/MU/mupolls-module/Resources/config/routing.yml").
```

**Comment by Guite** [comment]
Both fixed in eff8dd167f74d7785cb67623ba64e9198ffe5938
